tests/system/python/conftest.py

```diff
@@ -291,6 +291,9 @@ def pytest_addoption(parser):
                      help="Generic wait time between processes to run")
     parser.addoption("--retries", action="store", default=3, type=int,
                      help="Number of tries for polling")
+    # TODO: Temporary fixture, to be used with value False for environments where PI Web API is not stable
+    parser.addoption("--verify-north-data", action="store", default='true',
```

Comment:
we should change it to verify-north-interface, verify-north-data may be understood with locally stored data

Reply:
`action="store_false", default=True`